src/utils/address.ts

```diff
@@ -0,0 +1,19 @@
+import { getAddress } from 'viem';
+
+/**
+ * Renders the short address of a user.
+ *
+ * @param address - The address to render.
+ * @param chars - The number of characters to render.
+ * @returns The rendered short address.
+ */
+export function renderShortAddress(address?: string, chars = 5) {
```

Comment:
nit:

may be adding a parameter represent the head of str, instead of using the char+2 ?
such as 
`renderShortAddress(address?: string, head = 7, tail = 5) `

and also adding a checking
```
if (!address || address.length <= (head + tail)) {
   return address
}

```